VMEncryption/VMEncryption.pyproj

```diff
@@ -45,6 +45,9 @@
       <SubType>Code</SubType>
     </Compile>
     <Compile Include="main\check_util.py" />
+    <Compile Include="main\CryptMountConfigUtil.py">
+      <SubType>Code</SubType>
```

Comment:
><SubType>Code</SubType> [](start = 6, length = 23)

is this required?

Reply:
Not really, it just adds the file to the "project" for those using Visual Studio.

Reply:
ok. Others are not included so was checking. Maybe add it to all the files.

---
In reply to: [337153483](https://github.com/Azure/azure-linux-extensions/pull/916#discussion_r337153483) [](ancestors = 337153483)